hang_man.py

```diff
@@ -0,0 +1,158 @@
+            else:  # if dir already exists
+                word_file = urllib.request.urlretrieve(url, './word_data/words.txt')  # Download word file into dir
+                return os.path.relpath(word_file[0])  # Returns path
+        else:
+            return os.path.relpath('./word_data/words.txt')  # Returns path if file already exists
+
+    def assign_word(self):
+        empty_list = []
+        text_file = self.access_doc()           # Getting text file location from previous method
+        with open(text_file) as file:           # Opening text file
+            self.word_list = file.read().splitlines()       # Reading text files and splitting word
+            for word in self.word_list:         # Loop to iterate through all the words in file
+                if len(word) >= self.min_limit:         # Finding all words that are greater than or equal to min limit
+                    empty_list.append(word)         # All the words are being added to empty list
+                    right_word = random.choice(empty_list)      # From that list program randomly selects a random word
+            return right_word           # Returns the random word
+
+    def hangman_game(self):
+        correct_word = self.assign_word()       # Getting the word selected by program from previous method
+        self.right_list = list(correct_word)        # Converting the word into a list
+        right = 0                       # Counter to keep track of right word guessed
+        wrong = 0               # Counter to keep track of wrong word guessed
+        selected = []           # A list that keeps track of all the letter selected by user
+        random_string = ''.join(string.ascii_uppercase)     # Getting characters from A - Z
+        random_list = list(random_string)               # Converting the characters into list
+        temp_rlist = copy.deepcopy(random_list)         # making a temporary copy of the characters list
+
+        blank_list = ["__"] * len(self.right_list)      # A list is created with __ to show user status of word
+
+        while wrong != 6 and right != len(self.right_list):   # loop that executes until user guess right word/looses
+            print('For testing validation, right word is: ' + correct_word)  # Printing the right word for validation
+            print(self.hang_list[wrong])                            # Print the first stage of hangman
+            print('Pick from these letters: ' + ' '.join(random_list))  # Displays characters A-Z for user to pick from
+            print('Current status: ' + ' '.join(blank_list))        # Displays the current status of word by using __
+            guess_letter = input('Guess a letter: ').upper()        # Asks the user to input in one character
+            if not guess_letter.isalpha():                    # if the char is not string value it will provide an error
+                print('Please enter a string character')
```

Comment:
make this prompt more clear

Reply:
Changed wording for inputs and validation for better clarification